ads/_config.js

```diff
@@ -110,6 +110,7 @@ export const adConfig = {
     ],
     renderStartImplemented: true,
   },
+  adverticum: {},
```

Comment:
We decided to leave out any API for the time beeing. Maybe later we'll implement that too.